DataSetExplorer/DataSetSerializer/BetweenAnnotatorsSanityCheckExporter.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using CodeModel.CaDETModel.CodeItems;
+using DataSetExplorer.DataSetBuilder.Model;
+using OfficeOpenXml;
+
+namespace DataSetExplorer.DataSetSerializer
+{
+    class BetweenAnnotatorsSanityCheckExporter
```

Comment:
This name is interesting :)

What do the terms "BetweenAnnotators" and "SanityCheck" mean? We need terminology that can be understood by anyone familiar with the domain - I think that Koča used sanity check as a generic term, and betweenannotators isn't really a valid term. Ask the domain expert for a more formal definition to expand our ubiqutious language.